relbench/tasks/event.py

```diff
@@ -115,6 +123,9 @@ def make_table(self, db: Database, timestamps: "pd.Series[pd.Timestamp]") -> Tab
             """
         ).df()
 
+        if eval_timestamp_len == 1:
+            df = df[df.timestamp == df.timestamp.max()]
```

Comment:
what does this do?

Reply:
So we append 2 timestamps to the validation set and test set to construct the training data(because val set and test set only contains one timestamp originally). We then remove them to avoid having unnecessary timestamps